src/components/vm/nics/nicBody.jsx

```diff
@@ -200,29 +201,19 @@ export const NetworkTypeAndSourceRow = ({ idPrefix, onValueChanged, dialogValues
                                enableFlip={false}
                                bodyContent={
                                    <TextContent>
-                                       <Text component={TextVariants.h4}>
-                                           VEPA
-                                       </Text>
-                                       <Text component={TextVariants.p}>
-                                           {_("All VMs' packets are sent to the external bridge. Packets whose destination is a VM on the same host as where the packet originates from are sent back to the host by the VEPA capable bridge (today's bridges are typically not VEPA capable)")}.
-                                       </Text>
-                                       <Text component={TextVariants.h4}>
-                                           {_("Bridge")}
-                                       </Text>
                                        <Text component={TextVariants.p}>
-                                           {_("Packets whose destination is on the same host as where they originate from are directly delivered to the target macvtap device. Both origin and destination devices need to be in bridge mode for direct delivery. If either one of them is in \"VEPA\" mode, a VEPA capable bridge is required.")}
-                                       </Text>
-                                       <Text component={TextVariants.h4}>
-                                           {_("Private")}
-                                       </Text>
-                                       <Text component={TextVariants.p}>
-                                           {_("All packets are sent to the external bridge and will only be delivered to a target VM on the same host if they are sent through an external router or gateway and that device sends them back to the host. This procedure is followed if either the source or destination device is in \"Private\" mode.")}
-                                       </Text>
-                                       <Text component={TextVariants.h4}>
-                                           {_("Passthrough")}
+                                           {_("The mode influences the delivery of packets.")}
```

Comment:
I don't consider this change appropriate for z-stream. It is not a critical bug, and it adds new translatable strings. I know the previous commit does that too, but that at least has a better justification.

Reply:
Ah, oops -- sorry, I read the commits individually.